test/jdk/java/io/BufferedInputStream/TransferTo.java

```diff
@@ -0,0 +1,204 @@
+/*
+ * Copyright (c) 2021, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Hello Markus, the copyright year here should be `2022` instead of `2021`

Reply:
Actually I wrote this code in 2021.

Reply:
> Actually I wrote this code in 2021. What are the official rules here, do I have to use the original date or the latest change date?

I remember reading it somewhere but I can't seem  to locate the text right now and it isn't mentioned in the OpenJDK developer's guide either https://openjdk.org/guide/.
From what I remember, the copyright year is of the form `year1, [year2]` where `year1` is the year the file was first added to the project (so in this case it would be `2022`) and `year2` is the year it was last updated. That's my understanding of the guidelines. Others can correct me if that isn't accurate.

Reply:
Correct, so 2021 is right here, as the second date (last change) is *optional*.

Reply:
At least "added to the project" would be "merged", not "opened pr".

Reply:
Updated copyright to 2022 in https://github.com/openjdk/jdk/pull/6935/commits/a4b6f9130a045064370e49c0678429b26616fc3d as per your request.